runtime-modules/content/src/benchmarks/mod.rs

```diff
@@ -58,7 +63,14 @@ const fn gen_array_u128<const N: usize>(init: u128) -> [u128; N] {
 
     res
 }
-pub const DEFAULT_MEMBER_ID: u128 = 500;
+
+pub const BUY_NOW_PRICE: u32 = 100;
```

Comment:
Perhaps this can be dependent on the config values, so that it doesn't break if we change `min_starting_price` for example. Something like:
```rust
fn nft_buy_now_price<T: Config>() {
    Pallet::<T>::min_starting_price() + 1000u32.into()
}